cloudferry/lib/os/storage/cinder_storage.py

```diff
@@ -316,6 +316,9 @@ def create_volume_from_volume(self, volume, tenant_id):
 
         az = self.attr_override.get_attr(volume, 'availability_zone')
         volume_type = self.attr_override.get_attr(volume, 'volume_type')
+        if volume_type != volume['volume_type']:
+            LOG.debug("Volume type '%s' of volume '%s' is overridden for '%s'",
+                      volume['volume_type'], volume['id'], volume_type)
```

Comment:
Let's move this log to `attr_override.get_attr()` method, or remove it completely, it doesn't cover a lot of cases.